wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -201,6 +201,7 @@ namespace :wagn do
       Card::Cache.reset_global
       conn =  ActiveRecord::Base.connection
       # Correct time and user stamps
+      # USER related
```

Comment:
Next I want to create a clear separation of user stamps and authentication so we can plug this into an authentication system in the environment.